i18n/ko.json

```diff
@@ -1,265 +1,277 @@
+    "searchAndFilter": "검색 & 필터 설정",
     "searchFilters": "검색 필터",
     "settings": "설정",
-    "showHideText": "설정표시줄에 아이콘을 표시 / 숨기기",
-    "sortingFilters": "필터 및 정렬 옵션",
+    "showHideText": "설정 표시줄에 아이콘 표시/숨기기",
+    "sortingFilters": "필터 & 정렬 옵션",
     "statistics": "통계",
     "systemDefault": "시스템 기본값",
-    "tabFilter": "검색설정",
-    "tabMain": "기본설정",
+    "tabFilter": "검색 설정",
+    "tabMain": "기본 설정",
     "tabShortcuts": "단축키",
-    "tabView": "설정보기",
-    "thumbnailHeading": "썸네일뷰",
-    "variousSettings": "각종설정",
-    "viewSettings": "더 많은 뷰",
-    "wordCloud": "워드클라우드 및 최근에 열었던 파일",
+    "tabView": "보기 설정",
+    "thumbnailHeading": "섬네일 보기",
+    "variousSettings": "기타 설정",
+    "viewSettings": "더 많은 보기",
+    "wordCloud": "워드 클라우드 & 최근 열린 파일",
     "zoom": "확대 / 축소"
   },
   "SHORTCUTS": {
-    "andClickAnyFile": "그리고 디테일을 보려면 파일을 클릭하십시오.",
-    "andPress": "그리고 누르세요.",
-    "focusOnFile": "파일검색에 입력하세요.",
-    "focusOnMagic": "매직서치에 입력하세요.",
-    "hold": "누르고",
-    "makeLarger": "썸네일 크게 만들기",
-    "makeSmaller": "썸네일 작게 만들기",
-    "or": "또는",
-    "showClip": "클립뷰 보기",
-    "showDetail": "디테일뷰 보기",
-    "showFile": "텍스트뷰 보기",
-    "showFilm": "필름스트립뷰 보기",
-    "showFull": "풀뷰 보기",
-    "showThumb": "썸네일뷰 보기",
-    "shuffle": "현재결과 섞기",
-    "toggleAutoTags": "자동생성태그 보기",
-    "toggleDarkMode": "어두운 / 밝은모드 전환",
-    "toggleFileName": "파일이름, 해상도 및 길이 보기",
-    "toggleMinimalMode": "최소모드 전환 (모든패널 숨기기)",
-    "toggleSearchBar": "검색사이드바 전환",
-    "toggleSettings": "설정 열기",
-    "toggleWizard": "마법사 열기"
+    "andClickAnyFile": "파일을 클릭하여 자세히 보기",
+    "andPress": "누르기",
+    "focusOnFile": "파일 검색에 입력",
+    "focusOnFuzzy": "퍼지 검색에 입력",
+    "focusOnMagic": "마법 검색에 입력",
+    "hold": "Hold",
+    "makeLarger": "섬네일 크게",
+    "makeSmaller": "섬네일 작게",
+    "or": "or",
+    "showClip": "클립 보기",
+    "showDetail": "자세히 보기",
+    "showFile": "목록 보기",
+    "showFilm": "필름 스트립 보기",
+    "showFull": "전체 보기",
+    "showThumb": "섬네일 보기",
+    "shuffle": "현재 결과 섞기",
+    "toggleAutoTags": "자동 생성된 태그 표시",
+    "toggleDarkMode": "어두운 / 밝은 모드 전환",
+    "toggleFileName": "파일 이름, 해상도, 길이 표시",
+    "toggleMinimalMode": "최소 모드 전환(모든 패널 숨기기)",
+    "toggleSearchBar": "검색 사이드바 표시",
+    "toggleSettings": "설정 표시",
+    "toggleWizard": "마법사 표시"
   },
   "SIDEBAR": {
-    "and": "그리고",
-    "dupeByHash": "중복의심",
+    "and": "and",
+    "dupeByHash": "중복 의심",
     "dupeByLength": "비슷한 길이",
     "dupeBySize": "비슷한 크기",
-    "duplicate": "중복의심 :",
-    "exclude": "비디오가 없습니다.",
-    "file": "비디오 이름은",
-    "fileUnion": "비디오는",
-    "folder": "폴더 이름이",
-    "folderUnion": "폴더는",
+    "duplicate": "중복 의심:",
+    "exclude": "동영상 제외하기",
+    "file": "동영상 이름",
+    "fileUnion": "동영상 조합",
+    "folder": "폴더 이름",
+    "folderUnion": "폴더 조합",
     "found": "찾음",
-    "magic": "폴더 또는 비디오 이름",
-    "or": "또는",
-    "recentlyOpened": "최근에 생성한 허브 :",
-    "similarTo": "다음과 유사합니다.",
-    "sortAlphabetical": "알파벳순",
+    "fuzzy": "퍼지 검색",
```

Comment:
I have no problem with adding this line, but I don't think it will be used anywhere in the app at this time

Reply:
No problem. It's okay to keep using it